packages/common-common/src/api/extApiTypes.ts

```diff
@@ -160,4 +161,13 @@ export type PostRulesReq = {
 
 export type OnlyErrorResp = { error?: string | object };
 
+export type GetNewProfilesReq = {
+  jwt: string;
```

Comment:
Do we require the jwt = user login token for querying profiles? If it's exposed to the external API, we probably shouldn't, as that data is supposed to be used only on the commonwealth site.

Reply:
Will remove